website/docs/components/flyout/partials/code/component-api.md

```diff
@@ -31,68 +31,48 @@
 
 ### Contextual components
 
+!!! Warning
+
+The previous `Flyout::Header`, `Flyout::Description`, `Flyout::Body`, and `Flyout::Footer` subcomponents are now deprecated. View details in the [version history](/components/flyout?tab=version-history) on how to migrate to the equivalent [DialogPrimitive](/utilities/dialog-primitive?tab=code#component-api) subcomponents.
+
+!!!
+
 #### [F].Header
 
-The `Flyout::Header` component, yielded as contextual component.
+The `DialogPrimitive::Header` component, yielded as contextual component.
 
 <Doc::ComponentApi as |C|>
-  <C.Property @name="icon" @type="string">
-    [Icon](/icons/library) name.
-  </C.Property>
-  <C.Property @name="tagline" @type="string">
-    A string that helps the user maintain context when a Flyout is open.
-    <br/>This is **not** the title text, but a small piece of text above the title text.
-  </C.Property>
-  <C.Property @name="yield">
-    Elements passed as children are yielded as inner content of the "title" block.
-  </C.Property>
-  <C.Property @name="...attributes">
-    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  <C.Property>
+    It exposes the same API of the [`DialogPrimitive::Header`](/utilities/dialog-primitive?tab=code#dialogprimitiveheader) component.
```

Comment:
```suggestion
    It exposes the same API as the [`DialogPrimitive::Header`](/utilities/dialog-primitive?tab=code#dialogprimitiveheader) component.
```

Reply:
I've replaced this across all the occurrences in the documentation.